### PR TITLE
add mjml theme brienz

### PR DESCRIPTION
**Pull request by jos0405**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features" for all features, enhancements and bug fixes (until 3.3.0 is released) 
| Bug fix?                               | no
| New feature?                           | yes
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | no
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:
A theme for the grapesjs builder written in MJML, in order to test you need to be running the Mautic release candidate 3.3 or later.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. If you don't want to use Mautibox put these files into your theme directory (template upload doesn't work yet)

### CODE REVIEW

**Comment by alexhammerschmied** [comment]
mautibox throws an error while loading the pr. :(

**Comment by RCheesley** [comment]
> mautibox throws an error while loading the pr. :(

Probably needs to be updated to pull in the latest updates. Should hopefully work once I rebase which I will do in a few minutes!

File: themes/brienz/config.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Paprika",
```
Comment (RCheesley) [inline themes/brienz/config.json]:
```suggestion
  "name": "Brienz",
```

File: themes/brienz/html/email.mjml.twig [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+          <p><span style="font-size: 16px;"><a href="#" style="text-decoration: none; color: inherit;"><span style="font-weight: bold;"><span style="color: white;">Check out our blog:</span></span>
+            </a>
+          </p>
+          <p><span style="font-weight: bold;color: white;"><span style="font-size: 16px;"></span>
+          </p>
+          <p><span style="font-size: 28px;"><a href="#" style="text-decoration: none; color: inherit;"><span style="font-weight: bold;"><span style="color: white;">We are half way there!</span></span>
+            </a>
+            </span>
+          </p>
+          <p><span style="font-weight: bold;color:white;font-size: 27px;">See out progress!</span>
```
Comment (RCheesley) [inline themes/brienz/html/email.mjml.twig]:
```suggestion
          <p><span style="font-weight: bold;color:white;font-size: 27px;">See our progress!</span>
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
@jos0405 two small changes - the theme name needs updating:

screenshot-mautic32 ddev site-2021 02 22-11_31_29

Also a typo in the header text:

screenshot-mautic32 ddev site-2021 02 22-11_33_45
 
You should be able to click 'commit suggestion' to accept the proposed changes!

**Pull request merged**